modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/deal-notification-template/en-US.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+		<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,600;0,900;1,600;1,700&display=swap" rel="stylesheet" />
+		</head>
+
+		<body>
+			<div style="  background-color:#e6e6e6; padding: 20px;">
+			<main style="background-color: #ffffff;align-items: center; font-family: 'Source Sans Pro', sans-serif; border: solid 1px; border-color: #ffffff; ">
+					<h2 style="color: #1355cc; padding-left: 15px;">Partner Deal Registration</h2>
+
+					<table style="color:#565656;width: 100%; border-spacing: 15px; ;">
+						<tbody>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Partner:
+								</td>
+								<td>[%LEADNOTIFICATION_PARTNERACCOUNTNAME%]</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Partner Contact:
+								</td>
+								<td>
+									[%LEADNOTIFICATION_PRIMARYPROSPECTFIRSTNAME%]&nbsp;[%LEADNOTIFICATION_PRIMARYPROSPECTLASTNAME%]&nbsp;
+								</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Primary Contact Name:
+								</td>
+								<td>
+									[%LEADNOTIFICATION_PROSPECTACCOUNTNAME%]
+								</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Primary Contact Email:
+								</td>
+								<td>
+									[%LEADNOTIFICATION_PRIMARYPROSPECTEMAILADDRESS%]
+								</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Country:
+								</td>
+								<td>
+									[%LEADNOTIFICATION_PROSPECTCOUNTRYCODE%]
+								</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Account Name:
+								</td>
+								<td>[%LEADNOTIFICATION_CREATOR%]</td>
+							</tr>
+							<tr>
+							<td style=" font-weight: bold; white-space: nowrap; ">
+									Salesforce URL:&nbsp;
+								</td>
+								<td>
+								<a href="https://liferay.my.salesforce.com/[%LEADNOTIFICATION_LEADEXTERNALREFERENCECODE%]">https://liferay.my.salesforce.com/[%LEADNOTIFICATION_LEADEXTERNALREFERENCECODE%]</a>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+
+					<footer style="color:#565656;text-align: center">
+						Please claim the
+					<a href="https://liferay.my.salesforce.com/[%LEADNOTIFICATION_LEADEXTERNALREFERENCECODE%]">Lead</a>
+						and follow up with the prospect
+					</footer>
+				</main>
+			</div>
+		</div>
+		</body>
+
+</html>
```

Comment:
Hey @Manda-vit, the indentation looks weird.

Probably the sourceFormatter broke something. Try to fix it manually.